public/env.js

```diff
@@ -0,0 +1,4 @@
+window.env = {
```

Comment:
Tests are failing because they don't access env.js
I don't know if window is defined but we need to find a workaround, for example, a special env.js for tests.